src/psbt/mod.rs

```diff
@@ -249,8 +250,19 @@ impl<'psbt, Pk: MiniscriptKey + ToPublicKey> Satisfier<Pk> for PsbtInputSatisfie
     }
 
     fn check_older(&self, n: u32) -> bool {
-        let csv = self.psbt.global.unsigned_tx.input[self.index].sequence;
-        <Satisfier<Pk>>::check_older(&Older(csv), n)
+        let seq = self.psbt.global.unsigned_tx.input[self.index].sequence;
+        // https://github.com/bitcoin/bips/blob/master/bip-0112.mediawiki
+        // Disable flag set. return true
+        if n & SEQUENCE_LOCKTIME_DISABLE_FLAG != 0 {
+            true
```

Comment:
I don't see this in the BIP

Reply:
See the reference implementation and the c++ codebase.
We check the highest bit before doing any comparison checks.
```
    // To provide for future soft-fork extensibility, if the
    // operand has the disabled lock-time flag set,
    // CHECKSEQUENCEVERIFY behaves as a NOP.
    if ((nSequence & CTxIn::SEQUENCE_LOCKTIME_DISABLE_FLAG) != 0)
        break;

    // Compare the specified sequence number with the input.
    if (!checker.CheckSequence(nSequence))
        return set_error(serror, SCRIPT_ERR_UNSATISFIED_LOCKTIME);

    break;
```